docs/user-guide/getting-started.md

```diff
@@ -1,185 +1,205 @@
 ---
-    description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
+   description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
 ---
 # Quick Start Installation Guide
 
 In this guide, we'll walk you through installing and configuring the Ambassador Edge Stack in your Kubernetes cluster. Within a few minutes, your cluster will be routing HTTPS requests from the Internet to a backend service. You'll also have a sense of how the Ambassador Edge Stack is managed.
 
+Different options for installation include:
```

Comment:
I think we're still giving people too many options. In my opinion, the quickstart should show a single installation method (the default one we recommend) to get you up and running as quickly and easily. We can mention the other options, but including the full instructions for multiple methods is confusing. We do this on the current quickstart page.